projects/demo/src/modules/pipes/obfuscate/examples/1/index.ts

```diff
@@ -0,0 +1,20 @@
+import {Component} from '@angular/core';
+import {changeDetection} from '@demo/emulate/change-detection';
+import {encapsulation} from '@demo/emulate/encapsulation';
+import {tuiObfuscateOptionProvider, TuiObfuscatePipe} from '@taiga-ui/cdk';
+
+@Component({
+    standalone: true,
+    imports: [TuiObfuscatePipe],
+    templateUrl: './index.html',
+    encapsulation,
+    changeDetection,
+    providers: [
+        tuiObfuscateOptionProvider({
+            recipes: {
+                city: (value: string): string => 'x'.repeat(value.length),
```

Comment:
nit: can be `({ length }) => ...`. Also do we need both :strings? I think it's already typed since we pass it to the helper function?